README.md

```diff
@@ -477,6 +477,11 @@ __defaultOpacity__ -  Default opacity. Default is `0.6`
 
 __fadedOpacity__ - Hovered opacity. Default is `0.1`
 
+__additionalKeys__ - (array) declare additional fields to pass on context on event listener. E.g - 
```

Comment:
please make sure you add an example on how the data is available in event listener.

Reply:
@nxanil @natabal I tried out with this code, providing "hash" within additionalKeys, and below is the data available in the event listener on clicking a chord,

<img width="256" alt="Screen Shot 2019-09-17 at 5 55 18 PM" src="https://user-images.githubusercontent.com/24920919/65095121-bbb2c600-d974-11e9-95b1-1223a6430a18.png">

If I include "hash" as a listener parameter as well, it looks like this,

<img width="230" alt="Screen Shot 2019-09-17 at 6 03 41 PM" src="https://user-images.githubusercontent.com/24920919/65095336-8b1f5c00-d975-11e9-85e1-b1f4522e3311.png">

I am still not clear how our queries can consume this.